pkg/server/chaosd/redis.go

```diff
@@ -52,6 +52,16 @@ func (redisAttack) Attack(options core.AttackConfig, env Environment) error {
 
 	case core.RedisSentinelStopAction:
 		return env.Chaos.shutdownSentinelServer(attack, cli)
+
+	case core.RedisCachePenetrationAction:
+		pipe := cli.Pipeline()
+		for i := 0; i < attack.RequestNum; i++ {
+			pipe.Get(cli.Context(), "CHAOS_MESH_nqE3BWm7khHv")
```

Comment:
Just curious, what is this "CHAOS_MESH_nqE3BWm7khHv"?

Reply:
It's a meaningless code to make sure this key doesn't have a paired value.

Reply:
Oh I see, thanks.

Reply:
Why not use a random string?

Reply:
Because I think generating a random code is an unnecessary operation to simulate cache penetration. On the other hand, generating random code will bring additional time consumption which may make the loop runs slowly(especially when the request-num is very big). But I do accept the idea of using random codes, it's not a big deal.